sn_transfers/src/wallet/data_payments.rs

```diff
@@ -142,6 +147,37 @@ impl PaymentQuote {
         bytes
     }
 
+    /// Check self is signed by the claimed peer
```

Comment:
We could remove this comment and rename the function to 
"check_is_signed_by_claimed_peer"

Reply:
It's okay to push suggestions via github comments, it makes accepting them super easy (one click).
<img width="125" alt="Screenshot 2024-04-03 at 08 53 47" src="https://github.com/maidsafe/safe_network/assets/218554/51d79605-19cf-43ef-85f1-1b9f725326d0">